pkg/devicemapper/log.go

```diff
@@ -0,0 +1,12 @@
+package devicemapper
+
+// definitions from lvm2 lib/log/log.h
+const (
+	_              = iota
```

Comment:
iota starts at 0, which we don't need. But this shows me that we need it to start at 2 and go to 7. I'll make this clearer.